lib/node_modules/@stdlib/math/base/complex/inv/lib/cinv.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+var max = require( '@stdlib/math/base/special/max' );
+var FLOAT64_BIGGEST = require( '@stdlib/math/constants/float64-max' );
+var FLOAT64_SMALLEST = require( '@stdlib/math/constants/float64-smallest-normal' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+
+
+// VARIABLES //
+
+var LARGE_THRESHOLD = FLOAT64_BIGGEST * 0.5;
+var SMALL_THRESHOLD = FLOAT64_SMALLEST * ( 2.0 / EPS );
+var RECIP_EPS_SQR = 2.0 / ( EPS * EPS );
+
+
+// MAIN //
+
+/**
+* Compute the inverse of a complex number.
+*
+* @private
+* @param {(Array|TypedArray|Object)} [out] - output array
```

Comment:
Which is then also an issue [here](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/complex/divide/lib/cdiv.js#L26). Trivial to amend, but is it best if PRs target only one module at a time?